dataland-frontend/src/services/ApiClients.ts

```diff
@@ -47,6 +48,13 @@ export class ApiClientProvider {
     return new constructor(configuration, "/api-keys");
   }
 
```

Comment:
Let's clean this file up a little bit:

Let's give `getConstructedApi<T>(constructor)` another input "basePath: string" =>

`getConstructedApi(constructor, basePath: string)`

That way we save some boilerplate, since you don't need `getConstructedApiKeyManager()` and `getConstructedApiDocumentUpload()` anymore.

You can then for example do the following further down in the file:

`async getDocumentUploadController(): Promise<DocumentControllerApiInterface> {
    return this.getConstructedApi(DocumentControllerApi, "/documents");
  }`